src/components/scene.js

```diff
@@ -44,6 +44,20 @@ const INITIAL_VIEW_STATE = {
   bearing: 0
 };
 
+const TEXT_DATA = [
+  {
+    text: 'Hello\nWorld', // TODO make this an input and parse their str. Ex: new line becomes \n
+    position: [0, 0],
+    color: [255, 0, 0] // TODO temporarily red
+  }
+];
+
+const timestamp = new TextLayer({
+  data: TEXT_DATA,
+  getText: d => d.text,
+  getPosition: d => d.position, // TODO make this relative to camera
+  getColor: d => d.color,
+})
 
 const tileLayer = new TileLayer({
```

Comment:
@RaymondDashWu For #35 and reading Xiaoji's reply on visgl/deck.gl#4903 it looks like we're not defining these in the render function like we should. Try moving them down and see if your issue is resolved.

Reply:
Yup that fixed it! Finally!